joss_paper/paper.md

```diff
@@ -0,0 +1,176 @@
+---
+title: 'SunPy: A Python package for Solar Physics'
+tags:
+  - Python
+  - Astronomy
+  - Solar physics
+authors:
+  - name: Stuart J. Mumford
+    orcid: 0000-0003-4217-4642
+    affiliation: "1, 2"
+  - name: Nabil Freij
+  - name: Steven Christe
+  - name: Jack Ireland
+  - name: Florian Mayer
+  - name: Keith Hughitt
+  - name: Albert Y. Shih
+  - name: Daniel Ryan
+  - name: Simon Liedtke
+  - name: David Pérez-Suárez
+  - name: Pritish Chakraborty
+  - name: Vishnunarayan K I.
+  - name: Andrew Inglis
+  - name: Punyaslok Pattnaik
+  - name: Brigitta Sipocz
+  - name: Rishabh Sharma
+  - name: Andrew Leonard
+  - name: David Stansby
+  - name: Russell Hewett
+  - name: Alex Hamilton
+  - name: Laura Hayes
+  - name: Asish Panda
+  - name: Matt Earnshaw
+  - name: Nitin Choudhary
```

Comment:
orcid : 0000-0001-6915-4583
affiliation: 'Department of Mathematics, Indian Institute of Technology, Kharagpur, 721302, India'